src/core/sys/posix/sys/stat.d

```diff
@@ -1745,10 +1745,26 @@ else version (CRuntime_Musl)
             blksize_t st_blksize;
             blkcnt_t st_blocks;
 
-            timespec st_atim;
-            timespec st_mtim;
-            timespec st_ctim;
-            ino_t st_ino;
+            // Time64 on 32 bits
+            // See https://git.musl-libc.org/cgit/musl/commit/?id=38143339646a4ccce8afe298c34467767c899f51
+            static if (time_t.sizeof > c_long.sizeof)
+            {
+                private struct old_timespec {
+                    long tv_sec;
+                    long tv_nsec;
+                }
+
+                old_timespec __st_atim32;
```

Comment:
Shouldnt these be present for all 32-bit archs?

Also 
```
                timespec st_atim;
                timespec st_mtim;
                timespec st_ctim; 
```
These should not be in the else case, they are always needed.

Reply:
Yes